modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIM614AddDocumentationToAnAPIWithDocTypeSampleAndSDKThroughPublisherRestAPITestCase.java

```diff
@@ -204,205 +152,53 @@ public void testAddDocumentToAnAPISDKToFile() throws Exception {
-        String docUrl = "http://";
-        String filePathAPIM626 = TestConfigurationProvider.getResourceLocation() + File.separator +
-                "artifacts" + File.separator + "AM" + File.separator + "lifecycletest" +
-                File.separator + fileNameAPIM626;
-        String addDocUrl = publisherUrls.getWebAppURLHttp()+"publisher-old/site/blocks/documentation/ajax/docs.jag";
-
-        //Send Http Post request to add a new file
-        HttpPost httppost = new HttpPost(addDocUrl);
-        File file = new File(filePathAPIM626);
-        FileBody fileBody = new FileBody(file,"text/plain");
-
-        //Create multipart entity to upload file as multipart file
-        MultipartEntity multipartEntity = new MultipartEntity();
-        multipartEntity.addPart("docLocation", fileBody);
-        multipartEntity.addPart("mode",new StringBody(""));
-        multipartEntity.addPart("docName",new StringBody(docName));
-        multipartEntity.addPart("docUrl",new StringBody(docUrl));
-        multipartEntity.addPart("sourceType",new StringBody(sourceType));
-        multipartEntity.addPart("summary",new StringBody(summary));
-        multipartEntity.addPart("docType",new StringBody(docType));
-        multipartEntity.addPart("version",new StringBody(apiVersion));
-        multipartEntity.addPart("apiName",new StringBody(apiName));
-        multipartEntity.addPart("action",new StringBody("addDocumentation"));
-        multipartEntity.addPart("provider",new StringBody(apiProvider));
-        multipartEntity.addPart("mimeType",new StringBody(mimeType));
-        multipartEntity.addPart("optionsRadios",new StringBody(docType));
-        multipartEntity.addPart("optionsRadios1",new StringBody(sourceType));
-        multipartEntity.addPart("optionsRadios1",new StringBody(sourceType));
-
-        httppost.setEntity(multipartEntity);
-
-        //Upload created file and validate
-        HttpResponse response = httpClient.execute(httppost);
-        HttpEntity entity = response.getEntity();
-        JSONObject jsonObject1 = new JSONObject(EntityUtils.toString(entity));
-        assertFalse(jsonObject1.getBoolean("error"), "Error when adding files to the API ");
+        //Update Type and update document
+        documentDTO.setType(DocumentDTO.TypeEnum.SAMPLES);
+        documentDTO.setName(docName);
+        org.wso2.carbon.automation.test.utils.http.client.HttpResponse updateDocumentResponse =
+                restAPIPublisher.updateDocument(apiId, fileTypeDocumentId, documentDTO);
+        assertEquals(updateDocumentResponse.getResponseCode(), 200, "Error while update the documents");
+
+        //Update the document content
+        org.wso2.carbon.automation.test.utils.http.client.HttpResponse documentationResponse =
+                restAPIPublisher.updateContentDocument(apiId, fileTypeDocumentId, file);
+        assertEquals(documentationResponse.getResponseCode(), 200,
+                "Error while update documentation to API");
```

Comment:
The msg should be corrected as "Error while updating the documentation to the API". Please make sure you have used grammatically correct wordings since we are dealing with actual users.

Reply:
Fixed with b518b9f

Reply:
I think this has been missed. Shall we add this too?

Reply:
Done with b494ef3